homeassistant/components/blebox/light.py

```diff
@@ -0,0 +1,98 @@
+"""BleBox light entities implementation."""
+import logging
+
+from blebox_uniapi.error import BadOnValueError
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_HS_COLOR,
+    ATTR_WHITE_VALUE,
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_COLOR,
+    SUPPORT_WHITE_VALUE,
+    LightEntity,
+)
+from homeassistant.util.color import (
+    color_hs_to_RGB,
+    color_rgb_to_hex,
+    color_RGB_to_hs,
+    rgb_hex_to_rgb_list,
+)
+
+from . import BleBoxEntity, create_blebox_entities
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add):
```

Comment:
Our standard is to name the callback that adds entities `async_add_entities`.

Reply:
Fixed in https://github.com/home-assistant/core/pull/35786 - where I fixed it also in all other BleBox entity types.

(Along with other comments below).

Let me know if there's anything else I can improve.